cdap-cli/src/main/java/co/cask/cdap/cli/command/CreateDatasetInstanceCommand.java

```diff
@@ -52,21 +53,32 @@ public void perform(Arguments arguments, PrintStream output) throws Exception {
     String datasetName = arguments.get(ArgumentName.NEW_DATASET.toString());
     String datasetPropertiesString = arguments.getOptional(ArgumentName.DATASET_PROPERTIES.toString(), "");
     String datasetDescription = arguments.getOptional(ArgumentName.DATASET_DESCRIPTON.toString(), null);
+    String datasetOwner = arguments.getOptional(ArgumentName.OWNER_PRINCIPAL.toString(), null);
```

Comment:
Indeed I did not see this change... this is much better, however:
- does it have to be "owner principal"? Seems overly verbose. I think just "owner" is sufficient
- is this backward compatible with existing syntax?